tests/unit/src/test/scala/tests/codeactions/ActionableDiagnosticsSuite.scala

```diff
@@ -0,0 +1,42 @@
+package tests.codeactions
+
+import scala.meta.internal.metals.codeactions.ActionableDiagnostic
+
+import coursier.version.Version
+
+class ActionableDiagnosticsSuite
+    extends BaseCodeActionLspSuite("actionableDiagnostic") {
+
+  val oldOsLibVersion: Version = Version("0.7.8")
+
+  check(
+    "actionable-diagnostic-update",
+    s"""|//> <<>>using lib "com.lihaoyi::os-lib:${oldOsLibVersion.repr}"
+        |
+        |object Hello extends App {
+        |  println("Hello")
+        |}
+        |""".stripMargin,
+    ActionableDiagnostic.title,
+    expectedCode = "",
+    scalaCliOptions = List("--actions", "-S", scalaVersion),
+    expectNoDiagnostics = false,
+    scalaCliLayout = true,
+    saveResult = false,
```

Comment:
Why not save the results? That should be perfectly fine.

Reply:
Yes, I checked it again and it works with saving results.